src/nwb_datajoint/common/common_metrics.py

```diff
@@ -0,0 +1,153 @@
+from pathlib import Path
+import os
+import json
+import datajoint as dj
+import spikeinterface.toolkit as st
+import numpy as np
+    
+from .common_waveforms import Waveforms
+from .common_nwbfile import AnalysisNwbfile
+
+schema = dj.schema('common_metrics')
+
+@schema
+class MetricParameters(dj.Manual):
+    definition = """
+    # Parameters for computing quality metrics of sorted units
+    metric_params_name: varchar(200)
+    ---
+    metric_params: blob
+    """
+
+    def get_metrics_list(self):
+        "Returns a list of quality metrics available in spikeinterface"
+        # not all metrics in spikeinterface are supported yet
+        # metric_list = st.qualitymetrics.get_quality_metric_list()
+        metric_list = ['snr','isi_violation','nn_isolation','nn_noise_overlap']
+        return metric_list
+
+    def get_metric_default_params(self, metric: str):
+        "Returns default params for the given metric"
+        return _get_metric_default_params(metric)
+
+    def insert_default(self):
```

Comment:
`insert_default_params`

Reply:
changed to make it consistent with developer notes